### ISSUE TITLE
Upload to two different FTP servers

### ISSUE THREAD

**Issue by Gantron1** [git-ftp/git-ftp#276]
I have deployed my code on two different FTP servers. So far, I have used git-ftp successfully to upload to one of them. My question is, can I simply push to the other (using manual ftp login info)? Or will this mess something up?

**Comment by mkllnk** [comment]
You can push to the other one the same way as you did to the first one. They are treated separately. You can also define [scopes](https://github.com/git-ftp/git-ftp/blob/develop/man/git-ftp.1.md#scopes) to store the different login settings.

**Comment by vikaskhunteta** [comment]
@mkllnk Is this possible to push files on two or more locations with single push command?

**Comment by mkllnk** [comment]
No. You need two commands. But with scopes, you can do something like this:

```sh
git ftp push -s prod1 && git ftp push -s prod2
```

**Comment by LukasFritzeDev** [comment]
Since the character `,` is not allowed in scope / branch names (see [#525](https://github.com/git-ftp/git-ftp/pull/525/files)) we could implement a feature to set multiple scopes at once like this:
```
git ftp push -s prod1,prod2
```

This could be really handy if you have a hot fix and multiple environments to push this (staging and production for example).
